src/Dumbo.php

```diff
@@ -29,6 +29,20 @@ class Dumbo
     /** @var array<callable> */
     private $middleware = [];
 
+    /** @var array<string, mixed> Variables stored in the context */
+    private $configs = [];
+
+    public function set(string $key, $value): void
```

Comment:
Don't forget a doc block 👯  Same for `get`

Reply:
Ah yes!